addon/templates/components/x-select.hbs

```diff
@@ -15,7 +15,7 @@
       autofocus={{autofocus}}
       placeholder={{unless hasValues placeholder ''}}
       readonly={{unless canSearch 'readonly'}}
-
+      id={{if inputId inputId ''}}
```

Comment:
I'm guessing setting `id` to `null` or `undefined` will cause `html` not to render an `id`, in which case, just make it `id={{inputId}}`.